<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>635</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>635</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>635</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">	background: #202020; 
	color: white;
</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="insertChildAction">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #ededed;
background-color: #404040</string>
           </property>
           <property name="text">
            <string>+узел</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeRowAction">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #ededed;
background-color: #404040</string>
           </property>
           <property name="text">
            <string>-строка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="insertRowAction">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>color: #ededed;
background-color: #404040</string>
           </property>
           <property name="text">
            <string>+строка</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>715</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Allerta</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QTreeView {
    show-decoration-selected: 1;
}



QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
	
}

QTreeView{
	background: 	#F5F5F5;
	alternate-background-color: 	#ededed;
	color:	#4f4f4f ;
	font: 10pt;
	 border: 1px solid #567dbc;
}

QTreeView::branch:has-siblings:!adjoins-item {
    border-image: url(:/Branches/vline.png) 0;
}

QTreeView::branch:has-siblings:adjoins-item {
    border-image: url(:/Branches/branch-more.png) 0;
}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
    border-image: url(:/Branches/branch-end.png) 0;
}

QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
        border-image: none;
        image: url(:/Branches/branch-closed.png);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  {
        border-image: none;
        image: url(:/Branches/branch-open.png);
}


QTreeView::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;
}

QTreeView::item:selected {
    border: 1px solid #567dbc;
}

QTreeView::item:selected:active{
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);
}

QTreeView::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);
}


</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
	    color: #ededed;
    background-color: #404040
}

QMenuBar::item {
    spacing: 3px; /* spacing between menu bar items */
    padding: 1px 4px;
    background: transparent;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: #575757;
}

QMenuBar::item:pressed {
    background: #888888;
}</string>
   </property>
   <widget class="QMenu" name="menu_flie">
    <property name="styleSheet">
     <string notr="true">QMenu {
    color: #ededed;
	background: #353535;  /* sets background of the menu */
    border: 1px solid black;
}

QMenu::item {
    /* sets background of menu item. set this to something non-transparent
        if you want menu color and menu item color to be different */
    background-color: transparent;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background-color: #2F4F4F;
}</string>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_file"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuOther">
    <property name="styleSheet">
     <string notr="true">QMenu {
    color: #ededed;
	background: #353535;  /* sets background of the menu */
    border: 1px solid black;
}

QMenu::item {
    /* sets background of menu item. set this to something non-transparent
        if you want menu color and menu item color to be different */
    background-color: transparent;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background-color: #2F4F4F;
}</string>
    </property>
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="actionShow_chart"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="styleSheet">
     <string notr="true">QMenu {
    color: #ededed;
	background: #353535;  /* sets background of the menu */
    border: 1px solid black;
}

QMenu::item {
    /* sets background of menu item. set this to something non-transparent
        if you want menu color and menu item color to be different */
    background-color: transparent;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background-color: #2F4F4F;
}</string>
    </property>
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionLoad_file"/>
    <addaction name="actionUpload_file"/>
    <addaction name="actionConnect"/>
    <addaction name="actionLoad_http_file"/>
   </widget>
   <addaction name="menu_flie"/>
   <addaction name="menuOther"/>
   <addaction name="menuServer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">color: #ededed;
background-color: #404040</string>
   </property>
  </widget>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionShow_Chart">
   <property name="text">
    <string>Show Chart</string>
   </property>
  </action>
  <action name="actionShow_chart">
   <property name="text">
    <string>Show chart</string>
   </property>
  </action>
  <action name="actionNew_file">
   <property name="text">
    <string>&amp;New file</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
  <action name="actionUpload_file">
   <property name="text">
    <string>Upload file</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionLoad_http_file">
   <property name="text">
    <string>Load http file</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>&amp;Save as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
